src/alerts/slack.coffee

```diff
@@ -4,9 +4,7 @@ request = require 'request'
 module.exports = class SlackAlert extends Alert
   sendToSlack: (message) ->
     options =
-      url: "#{@config.host}/services/hooks/incoming-webhook"
-      qs:
-        token: @config.token
+      url: "https://hooks.slack.com/services/#{config.token}"
```

Comment:
The url listed in the curl command at the bottom of [the documentation page](https://api.slack.com/incoming-webhooks) includes what looks like three url parameters - `T00000000 `, `B00000000 `, and finally the token `XXXXXXXXXXXXXXXXXXXXXXXX `. All three should likely be separate config properties. I'm not sure what the first two correspond to and the documentation doesn't specify, but it would be good to double check that.